rts/at/bridge/src/main/java/org/jboss/narayana/rest/bridge/inbound/InboundBridge.java

```diff
@@ -217,7 +218,8 @@ private Transaction getTransaction() {
         final Transaction transaction;
 
         try {
-            transaction = SubordinationManager.getTransactionImporter().importTransaction(xid);
+            transaction = SubordinationManager.getTransactionImporter()
+                    .importTransaction(xid, TxControl.getDefaultTimeout());
```

Comment:
The timeout on the bridged JTA transaction should be set to never time out since its lifecycle is under the control of the REST-AT transaction